web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -213,15 +211,15 @@ const TeamReviews = ({ onBack, periodId }) => {
   const loadTeamMembers = () => {
     let members = [];
 
-    if (approvalMode) {
+    if (!approvalMode || (isAdmin && showAll)) {
```

Comment:
When the review period is not in the AWAITING_APPROVAL state OR
when the logged user is an admin AND they want to show all
then show all the assignments.
Otherwise filter the assignments to only those for members that report to you.